api/v1alpha2/provider_types.go

```diff
@@ -65,6 +65,17 @@ type ProviderSpec struct {
 	// namespace of the provider will be used. There is no validation of the yaml content inside the configmap.
 	// +optional
 	AdditionalManifestsRef *ConfigmapReference `json:"additionalManifests,omitempty"`
+
+	// ManifestPatches are applied to rendered provider manifests to customize the
+	// provider manifests. Patches are applied in the order they are specified.
+	// The `kind` field must match the target object, and
+	// if `apiVersion` is specified it will only be applied to matching objects.
+	//
+	// This should be an inline yaml blob-string
+	//
+	// https://tools.ietf.org/html/rfc7386
```

Comment:
I am not sure how important this information is, however, having them in the generated CRD manifests seems uncommon at least to me:

```
\n This should be an inline yaml blob-string
                  \n https://tools.ietf.org/html/rfc7386"

```